Libraries/Text/Text.js

```diff
@@ -64,9 +66,15 @@ const Text: React.AbstractComponent<
             onPress,
             onPressIn(event) {
               setHighlighted(!suppressHighlighting);
+              if (onPressIn) {
```

Comment:
Can you add `onPressIn` and `onPressOut` in the dependencies below.

Reply:
Woops, sorry about that. It's good now.